dashboard/test/models/script_test.rb

```diff
@@ -2697,68 +1909,6 @@ def populate_cache_and_disconnect_db
     assert_equal expected_levels, actual_levels
   end
 
-  test 'accessing lessons through lesson groups is same as directly from unit' do
-    l1 = create :level
-    l2 = create :level
-    l3 = create :level
-    l4 = create :level
-    dsl = <<-UNIT
-      lesson_group 'content', display_name: 'Content'
-      lesson 'Lesson1', display_name: 'Lesson1'
-      level '#{l1.name}'
-      lesson 'Lesson2', display_name: 'Lesson2'
-      level '#{l2.name}'
-      level '#{l3.name}'
-      lesson_group 'content2', display_name: 'Content'
-      lesson 'Lesson3', display_name: 'Lesson3'
-      level '#{l4.name}'
-    UNIT
-
-    unit = Script.add_unit(
-      {name: 'lesson-group-test-script'},
-      ScriptDSL.parse(dsl, 'a filename')[0][:lesson_groups]
-    )
-
-    assert_equal unit.lesson_groups[0].lessons[0], unit.lessons[0]
-    assert_equal unit.lesson_groups[0].lessons[0].absolute_position, 1
-    assert_equal unit.lesson_groups[0].lessons[1], unit.lessons[1]
-    assert_equal unit.lesson_groups[0].lessons[1].absolute_position, 2
-    assert_equal unit.lesson_groups[1].lessons[0], unit.lessons[2]
-    assert_equal unit.lesson_groups[1].lessons[0].absolute_position, 3
-  end
-
-  test 'level can only be added once to a lesson' do
```

Comment:
overall, I agree with your above comment, though this test seems like it may test more than the seeding logic, in particular that a lesson can't have duplicate levels

Reply:
I think you are right about this one. while I didn't perform an exhaustive audit here, I did happen do do some work in this area on the migrated unit path, and a couple of tests on that codepath have already been added in this space: https://github.com/code-dot-org/code-dot-org/pull/43423